src/stories/Radio&Checkbox.stories.tsx

```diff
@@ -0,0 +1,47 @@
+import React, { useState } from 'react';
+import RadioCheckboxGroup from 'components/RadioButton';
+import { milkList } from 'assets/sampleBobaAPI';
```

Comment:
import your test data inside stories.
Otherwise, the tests will break when our sampleAPI is removed in the future